docs/release-notes/version-2.0.md

```diff
@@ -0,0 +1,48 @@
+# ntc-netbox-plugin-onboarding v2.0 Release Notes
+
+## v2.0
+
+### New Features
+
+#### NetBox 2.9 support
+
+Support for NetBox 2.9 has been added, while keeping legacy release 2.8 also supported.
+
+#### Onboarding Extensions
+
+Allows to provide user's own Python module to customize onboarding process. This feature allows to onboard stack devices, HA devices, clustered devices and more customization. Example code is located in `docs/examples/example_ios_set_device_role.py` file and shows how to assign device roles automatically.
+
+#### Onboarding details exposed in a device view
+
+Date, Status, Last success and Latest task id related to the onboarded device are presented in under the device view.
+
+#### Onboarding task view
+
+Onboarding details exposed in a dedicated view, including NetBox's ChangeLog.
+
+#### Onboarding Changelog
+
+Onboarding uses NetBox's ChangeLog to display user and changes made to the Onboarding Task object.
+
+#### Skip onboarding feature
+
+New attribute in the OnboardingDevice model allows for skipping the onboarding request on devices with disabled onboarding setting.
+
+#### Improved logging
+
+Logging has been improved and refactored.
+
+#### Onboarding process refactor
+
+Onboarding process has been completely refactored, including classes disaggregation.
```

Comment:
All these separate sections make this pretty verbose - personally I'd rather follow the [NetBox release note format](https://netbox.readthedocs.io/en/stable/release-notes/version-2.9/) and just have bulleted lists under "Enhancements" and "Bug Fixes" except where a major change requires a more in-depth (paragraph-length) description.

Reply:
Did a shorter version